src/it/missing-index/verify.groovy

```diff
@@ -17,6 +17,6 @@
  * under the License.
  */
 
-assert new File(basedir, 'build.log').getText('UTF-8').contains("create src/main/resources/index.jelly:");
+assert new File(basedir, 'build.log').getText('UTF-8').contains("A default one will be created using the description of the pom.xml");
```

Comment:
More interesting to assert text of the generated file.

Reply:
Yes indeed. I updated the test to also include an assertion on the file's content